File: THCS/2021_2022/DaNang_2122/Bai3/TAMGIAC.py
# Đọc dữ liệu từ file
with open('TAMGIAC.INP', 'r') as file:
    a, b = map(int, file.readline().split())

# Tính khoảng giá trị của c
min_c = abs(a - b) + 1  # c phải lớn hơn |a - b|
max_c = a + b - 1       # c phải nhỏ hơn a + b

# Số lượng các giá trị c thỏa mãn
count_c = 0
if min_c <= max_c:
    count_c = max_c - min_c + 1

# Ghi kết quả ra file
with open('TAMGIAC.OUT', 'w') as file:
    file.write(str(count_c) + '\n')

File: THCS/2021_2022/DaNang_2122/Bai3/TAMGIAC.cpp
#include <iostream>
#include <fstream>
#include <cmath> // Thư viện toán học để sử dụng hàm abs (giá trị tuyệt đối)
using namespace std;

int main() {
    ifstream fin("TAMGIAC.INP");
    ofstream fout("TAMGIAC.OUT");

    int a, b;
    fin >> a >> b;

    // Tính khoảng giá trị của c
    int min_c = abs(a - b) + 1;  // c phải lớn hơn |a - b|
    int max_c = a + b - 1;       // c phải nhỏ hơn a + b

    // Số lượng các giá trị c thỏa mãn
    int count_c = 0;
    if (min_c <= max_c) {
        count_c = max_c - min_c + 1;
    }

    // Ghi kết quả ra file
    fout << count_c << endl;

    fin.close();
    fout.close();

    return 0;
}
